Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang
+
+ENV PORT 80
+EXPOSE $PORT
+
+RUN echo "\
+    UserKnownHostsFile /etc/secret-volume/known_hosts\n\
+    IdentityFile /etc/secret-volume/id_rsa\n\
+" >> /etc/ssh/ssh_config
+
+RUN git config --global user.name 'github-review-helper'
```

Comment:
https://github.com/salemove/github-review-helper/pull/19/commits/db7b6f03bd51bbbf0152c896e5fdf1f65e26f232. Originally thought that there's no point in doing so, but after some reading up it seems reasonable, for the added efficiency.